### PR TITLE
Add IPFS max size check

### PR DESCRIPTION
**Pull request by sparkplug0025**
Completes #198

### CODE REVIEW

File: packages/webapp/src/_api/handlers/ipfs-upload-handler.ts [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+interface ValidUploadActions {
+    [contract: string]: {
+        [action: string]: { maxSize: number };
+    };
+}
+
+const VALID_UPLOAD_ACTIONS: ValidUploadActions = {
+    [edenContractAccount]: {
+        inductprofil: { maxSize: 1_000_000 },
+        inductvideo: { maxSize: 100_000_000 },
```
Comment (brandonfancher) [inline packages/webapp/src/_api/handlers/ipfs-upload-handler.ts:22]: Should we move these into the config?

File: packages/webapp/src/_app/hooks/forms.ts [hunk 2]
```diff
@@ -42,5 +43,15 @@ export const handleFileChange = async (
         return onError(new Error(`You can only select ${mimePrefix} files`));
     }
 
+    if (file.size > maxSize) {
+        return onError(
+            new Error(
+                `The file cannot be larger than ${Math.floor(
+                    maxSize / 1_000_000
```
Comment (brandonfancher) [inline packages/webapp/src/_app/hooks/forms.ts:50]: Here too.

File: packages/webapp/src/inductions/components/induction-profile-form.tsx [hunk 3]
```diff
@@ -97,7 +97,12 @@ export const InductionProfileForm = ({
                     accept="image/*"
                     label="select an image file"
                     onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-                        handleFileChange(e, "image", setUploadedImage)
+                        handleFileChange(
+                            e,
+                            "image",
+                            1_000_000,
```
Comment (brandonfancher) [inline packages/webapp/src/inductions/components/induction-profile-form.tsx:103]: Here too.

File: packages/webapp/src/_api/handlers/ipfs-upload-handler.ts (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/_api/handlers/ipfs-upload-handler.ts:22]: ...and reference them in the three places they're also used below?

Comment (sparkplug0025) [reply packages/webapp/src/_api/handlers/ipfs-upload-handler.ts]: good idea! done!

**Review by brandonfancher — APPROVED** [review]

**Pull request merged**